share/availability/light/availability_test.go

```diff
@@ -294,17 +296,23 @@ func TestSharesRoundTrip(t *testing.T) {
 				}
 			}
 
-			namespacedShares, _, _ := b.Data.ComputeShares(uint64(0))
+			// TODO: set useShareIndexes to true. This requires updating the
+			// transaction data in this test to include share indexes.
+			shares, err := appshares.Split(b.Data, false)
```

Comment:
I'm fine making an issue for this but we should do it sooner rather than later to keep tests up to date.

Reply:
We can put true there already btw. There is no transaction logic involved in the test

Reply:
Well, the logic that we are testing here does not have anything to do with the content of the shares. It just passes shares back and forth and disconnected from the App's share verification logic so I would put true here already.

Reply:
Putting `true` here will fail these unit tests because [this error](https://github.com/celestiaorg/celestia-app/blob/906fdf74f871f13d3fe0c632dfda95db494521ee/pkg/shares/share_splitting.go#L130)

Reply:
Ok, so we cannot even split shares, unless we generate them with some special logic.

Reply:
> I'm fine making an issue for this

I opened https://github.com/celestiaorg/celestia-node/issues/1231 if reviewers are comfortable merging this PR as-is and following-up with that issue

Reply:
I'm fine with that.

Reply:
+